Assets/Scripts/SS3D/Systems/Tile/Connections/TileObjectType.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace SS3D.Systems.Tile
+{
+    public enum TileObjectGenericType
```

Comment:
Make a summary to explain what those enums are used for. Make one file for each enum.